pyodide-build/requirements.txt

```diff
@@ -0,0 +1,9 @@
+build==0.7.0
```

Comment:
We don't need to have a requirements.txt file if they are already specified in setup.cfg IMO.
I'm not sure we really need to pin requirements at the moment. At the very least we could use semver compatible versions e.g. `build~=0.7.0` that would install 0.7.x but not 0.8.0.

Reply:
Well we pretty aggressively use private features of `build`, basically in order to minimize the complexity of PR #2272. We probably should be pinning this exact version until we fix that.

Reply:
For build -- fine. I meant all the other packages there as well.

Reply:
I added this for testing in CI, but yeah, we probably don't need to pin versions (except for `build`).